component/src/main/java/me/androidbox/component/agenda/AgendaDetailTopBar.kt

```diff
@@ -0,0 +1,124 @@
+package me.androidbox.component.agenda
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import me.androidbox.component.R
+import me.androidbox.component.ui.theme.BusbyTaskyTheme
+import me.androidbox.component.ui.theme.backgroundBackColor
+import me.androidbox.component.ui.theme.fontWhiteColor
+import me.androidbox.component.ui.theme.topbarFontColor
+
+@Composable
+fun AgendaDetailTopBar(
+    editModeType: EditModeType,
+    displayDate: String,
+    modifier: Modifier = Modifier,
+    onCloseClicked: () -> Unit,
+    onEditClicked: () -> Unit,
+    onSaveClicked: () -> Unit
+) {
+    Row(modifier = modifier,
+        verticalAlignment = Alignment.CenterVertically,
+    horizontalArrangement = Arrangement.SpaceBetween) {
+        IconButton(onClick = {
+            onCloseClicked()
+        }) {
+            Icon(painterResource(id = R.drawable.close_white),
+                tint = Color.White,
```

Comment:
@philipplackner @MKiperszmid Just wondering if this icon is white, why do we need to set the tint color. Without it the icon color is black

Reply:
Sometimes when importing icons on the drawable folder, it defaults the color to black
Maybe that's the case?

Reply:
Yes I think if your device is in light mode, it defaults to black, so it looks good on a light surface